responses.py

```diff
@@ -692,6 +692,23 @@ def stop(self, allow_assert=True):
                 )
             )
 
+    def assert_call_count(self, url, count):
+        call_count = len(
+            [
+                1
+                for call in self.calls
+                if call.request.url == _ensure_url_default_path(url)
+            ]
+        )
+        if call_count == count:
+            return True
+        else:
+            raise AssertionError(
+                "Expected request to be called {0} times. Called {1} times.".format(
```

Comment:
Could this include the URL that was provided? It could be helpful in figuring out which test is failing if assertions are made in a loop.

Reply:
URL included.